### PR TITLE
[WIP] OMS front-end

### PR DESCRIPTION
**Pull request by HuangShengYao108590061**
[WIP] OMS front-end, this PR is still on the work in progress status ( don't merge it first)

### CODE REVIEW

**Review by NTUT-LUNA — APPROVED** [review]

File: app/controllers/orderController.py [hunk 1]
```diff
@@ -11,17 +11,34 @@ def index():
...
         return redirect(url_for('index.index'))
 
-    return render_template('order/orderDetails.html')
+    order = Order.getByID(order_id)
+
+    orders = Order.getAll()
+    form = NewProductForm(orders)
+
+    form.initProductData(order)
+    return render_template('order/manageOrder.html', form=form, order=order)
```
Comment (Ming119) [inline app/controllers/orderController.py:36]: I am wondering what is the purpose of new product form

**Review by Ming119 — COMMENTED** [review]
Please remove WIP after finish the PR

**Review by Ming119 — CHANGES_REQUESTED** [review]
@HuangShengYao108590061 please rebase this branch to 108590050-Ming so that the non-PR backend changes of OMS can be see on your branch

File: app/controllers/orderController.py [hunk 2]
```diff
@@ -11,17 +11,34 @@ def index():
         flash(f'You are not allowed to access.', 'danger')
         return redirect(url_for('index.index'))
 
-    return render_template('order/manageOrder.html')
+    searchForm = SearchForm()
+
+    return render_template('order/manageOrder.html',
+                            searchForm    = searchForm,
+                            filter        = type
```
Comment (Ming119) [inline app/controllers/orderController.py:18]: filter is only for filterIndex

**Review by Ming119 — CHANGES_REQUESTED** [review]

**Pull request merged**